cluster-scope/base/nvidia.com/clusterpolicy/gpu-cluster-policy/gpu-cluster-policy.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: nvidia.com/v1
+kind: ClusterPolicy
+metadata:
+  name: gpu-cluster-policy
+spec:
+  dcgmExporter:
+    affinity: {}
```

Comment:
Do you know if we have to specify all these empty attributes?

Reply:
I do not know.  I got this all from the web UI. I couldn't find documentation about it.
The only attribute that I am explicitly setting here is `args` (set to `[]`) for dcgmExporter.